uaclient/system.py

```diff
@@ -154,28 +156,33 @@ def get_platform_info() -> Dict[str, str]:
         "type": "Linux",
     }
 
-    version = os_release["VERSION"]
-    # Strip off an LTS point release (20.04.1 LTS -> 20.04 LTS)
-    version = re.sub(r"\.\d LTS", " LTS", version)
+    version = re.sub(r"\.\d LTS", " LTS", os_release["VERSION"])
```

Comment:
while we're here lets be more defensive on this one too
```diff
-version = re.sub(r"\.\d LTS", " LTS", os_release["VERSION"])
+version = re.sub(r"\.\d LTS", " LTS", os_release.get("VERSION", ""))
```

Same down below where we also use `os_release["VERSION"]`

Reply:
solved via code